WordPress/openverse-frontend
# Make tabs closable

## Fixes


Fixes #1835 by @obulat

## Description


This PR adds a Close button to `VTabs` component for the new mobile modal implementation:
<img width="169" alt="Screen Shot 2022-09-22 at 5 28 01 PM" src="https://user-images.githubusercontent.com/15233243/191774611-72ee0755-3c12-47ab-88dc-4d8bad3d04e2.png">

## Testing Instructions

<img width="492" alt="Screen Shot 2022-09-22 at 5 24 05 PM" src="https://user-images.githubusercontent.com/15233243/191774632-4aaf73c6-0256-4ed0-99f3-83008a7d59c0.png">
Run `pnpm storybook` and look at the VTabs story. The closable story should have a cross Close button, and on clicking it, you should see a `close` event in the Actions tab of storybook:

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
